techniques/systemSettings/misc/variableFromJsonFile/1.0/variableFromJsonFile.st

```diff
@@ -0,0 +1,50 @@
+#####################################################################################
+# Copyright 2015 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+bundle agent variable_from_json_file
+{
+  vars:
+    "json_file"        string => "&JSON_FILE&";
+    "namespace"        string => "&NAMESPACE&";
+    "name"             string => "&VARIABLE_NAME&";
+    "directory_server" string => "&SHARED_FILES_FOLDER&"; # shared files folder
```

Comment:
since it is a multivalued technique, with multiple variables, the values up there will be lists, so you should use arrays - otherwise the values here will contain a comma-separated list of filename, namespace and name